### PR TITLE
*: replace black-white-list by table filter

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- Add a doc for the "table filter" feature shared by all new TiDB ecosystem tools
- Update related contents for BR and TiDB Lightning

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3065
- Other reference link(s):

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -195,6 +195,23 @@ br backup table \
 
 备份期间有进度条在终端中显示。当进度条前进到 100% 时，说明备份已完成。在完成备份后，BR 为了确保数据安全性，还会校验备份数据。
 
+### 使用表库过滤备份数据
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:198]:
```suggestion
### 使用表库过滤备份多张表的数据
```

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -195,6 +195,23 @@ br backup table \
...
+
+如果你需要以更复杂的过滤条件来备份多个表，执行 `br backup full` 命令，并使用 `--filter` 或 `-f` 来指定使用[表库过滤](/table-filter.md)。
```
Comment (kennytm) [inline br/backup-and-restore-tool.md:200]: consecutive use of "使用" 🙃 could we reword the last clause?

File: table-filter.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    ./tidb-lightning -f 'foo*.*' -f 'bar*.*' -d /tmp/data/ --backend tidb
+    #                ^~~~~~~~~~~~~~~~~~~~~~~
+    ```
+
+### TOML 配置文件
+
+在 TOML 文件中，表库过滤规则以[字符串数组](https://toml.io/en/v1.0.0-rc.1#section-15)的形式指定。以下为各个工具中的使用示例：
```
Comment (kennytm) [inline table-filter.md:51]:
```suggestion
在 TOML 文件中，表库过滤规则以[字符串数组](https://toml.io/cn/v1.0.0-rc.1#%E6%95%B0%E7%BB%84)的形式指定。以下为各个工具中的使用示例：
```

File: table-filter.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+|-----------------------|--------|--------|--------|------------------|
+| irrelevant.table      |        |        |        | 默认（拒绝） |
+| employees.employees   | ✓      |        |        | 规则 1（接受）  |
+| employees.dept_emp    | ✓      | ✓      |        | 规则 2（拒绝）  |
+| employees.departments | ✓      | ✓      | ✓      | 规则 3（接受）  |
+| else.departments      |        | ✓      | ✓      | 规则 3（接受）  |
+
+> **注意：**
+>
+> 在 TiDB 生态工具中，无论表库过滤如何设置，系统表总是被排除。系统表有以下六个：
```
Comment (kennytm) [inline table-filter.md:245]:
```suggestion
> 在 TiDB 生态工具中，无论表库过滤如何设置，系统库总是被排除。系统库有以下六个：
```

File: br/backup-and-restore-tool.md (hunk 2 above)
Comment (ran-huang) [reply br/backup-and-restore-tool.md:200]: Good catch! Updated. PTAL.

File: br/backup-and-restore-tool.md [hunk 5]
```diff
@@ -195,6 +195,23 @@ br backup table \
 
 备份期间有进度条在终端中显示。当进度条前进到 100% 时，说明备份已完成。在完成备份后，BR 为了确保数据安全性，还会校验备份数据。
 
+### 使用表库过滤备份多张表的数据
+
+如果你需要以更复杂的过滤条件来备份多个表，执行 `br backup full` 命令，并使用 `--filter` 或 `-f` 来指定使用[表库过滤](/table-filter.md)。
```
Comment (ran-huang) [inline br/backup-and-restore-tool.md:200]:
```suggestion
如果你需要以更复杂的过滤条件来备份多个表，执行 `br backup full` 命令，并使用 `--filter` 或 `-f` 来指定[表库过滤](/table-filter.md)规则。
```

Comment (kennytm) [reply br/backup-and-restore-tool.md:200]: LGTM

**Comment by overvenus** [comment]
Please update filter usage in CDC too. https://docs.pingcap.com/zh/tidb/stable/manage-ticdc#%E5%90%8C%E6%AD%A5%E4%BB%BB%E5%8A%A1%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%E6%8F%8F%E8%BF%B0

**Comment by ran-huang** [comment]
> Please update filter usage in CDC too. https://docs.pingcap.com/zh/tidb/stable/manage-ticdc#%E5%90%8C%E6%AD%A5%E4%BB%BB%E5%8A%A1%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%E6%8F%8F%E8%BF%B0

@overvenus Thanks! Updated in https://github.com/pingcap/docs-cn/pull/3916/commits/92a5fd454c31e323bd8e049fffb6406f0ebd3ec7

**Comment by yikeke** [comment]
Please help review this PR, thanks~ @TomShawn

File: br/backup-and-restore-tool.md [hunk 6]
```diff
@@ -195,6 +195,23 @@ br backup table \
 
 备份期间有进度条在终端中显示。当进度条前进到 100% 时，说明备份已完成。在完成备份后，BR 为了确保数据安全性，还会校验备份数据。
 
+### 使用表库过滤备份多张表的数据
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:198]:
```suggestion
### 使用表库过滤功能备份多张表的数据
```

File: br/backup-and-restore-tool.md [hunk 7]
```diff
@@ -358,6 +375,22 @@ br restore table \
     --log-file restorefull.log
 ```
 
+### 使用表库过滤恢复数据
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:378]:
```suggestion
### 使用表库功能过滤恢复数据
```

**Comment by ran-huang** [comment]
@TomShawn PTAL again, thanks~

File: table-filter.md [hunk 8]
```diff
@@ -0,0 +1,252 @@
+---
+title: 表库过滤
+summary: 在 TiDB 生态工具中使用表库过滤功能。
+category: reference
```
Comment (TomShawn) [inline table-filter.md:4]:
```suggestion
```

**Pull request merged**